utils/create_dummy_models.py

```diff
@@ -94,8 +96,12 @@
     "TFCamembertModel",
     "TFCamembertForCausalLM",
     "DecisionTransformerModel",
+    "GraphormerModel",
+    "InformerModel",
     "JukeboxModel",
     "MarianForCausalLM",
+    "MaskFormerSwinModel",
+    "MaskFormerSwinBackbone",
```

Comment:
processor types are not included in XXX_MAPPING_NAMES (which is intended, told by Niels)